cg/meta/workflow/analysis.py

```diff
@@ -705,6 +705,17 @@ def _write_panel(out_dir: Path, content: list[str]) -> None:
             file_path=Path(out_dir, ScoutExportFileName.PANELS),
         )
 
+    @staticmethod
+    def _write_panel_as_tsv(out_dir: Path, content: list[str]) -> None:
```

Comment:
lets try to put these in the Nallo API

Reply:
Nice, that worked.
![image](https://github.com/user-attachments/assets/420388ee-81a1-426f-a1a2-bd1b6609aff8)